app-text/xapers/xapers-0.8.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
copyright line is obsolete, we have the new one

Reply:
Fixed.